### ISSUE TITLE
use green bar in the chart instead of the gray one

### ISSUE THREAD

**Issue by gillbates** [statping-ng/statping-ng#158]
Hi Team,

![iShot_2022-08-01_14 40 38](https://user-images.githubusercontent.com/783547/182087818-c6b9f857-81e3-4c6e-a476-16bfef99af80.jpg)

Currently we are using red bar in the chart for failures, and gray box for normal ones. Can we use green bar instead of gray one to represent "Ok" status? This will then remain consistent UI style with the chart below.

**Comment by sndrr** [comment]
You can create your own theme by copying the statping scss files from the github source and edit those.

### How to do so?

1. Go to the dashboard > settings > theme
2. Enable the local assets
3. Fill the textboxes with their corresponding content (see `frontend/src/assets/scss` for stock content).
4. Edit as you like
![image](https://user-images.githubusercontent.com/5821198/182563770-f9c98658-bf55-48c6-80a1-a76c000e2e9b.png)
5. Save the contents. That will trigger a recompile of the CSS files. See this log entry:
![image](https://user-images.githubusercontent.com/5821198/182568125-6833073b-fbe5-49cc-a494-ea978ffa1dbb.png)

### Alternative
There are other ways to get the content there, but they assume you have some tooling installed.

- Assuming you are using Docker
- have the package `jq` installed
- you have this github repo checked out and are in its root

1. Find the mount of your statping volume
```
docker inspect statping | jq -r .[].Mounts[].Source
```
Example (this is only a part of `docker inspect statping`:
![image](https://user-images.githubusercontent.com/5821198/182568912-d9dd3c16-c2b5-4b1e-9fed-ec79ba40f66c.png)

1. Copy `frontend/src/assets/scss` to the docker volume:
```
sudo cp -R frontend/src/assets/scss /path/to/docker/statping/volume/assets
```
2. Copy the `frontend/src/assets/css` to the docker volume:
```
sudo cp -R frontend/src/assets/css /path/to/docker/statping/volume/assets
```
3. Edit away in the dashboard (see above from step 4 onwards).

**Comment by gillbates** [comment]
thx @sndrr 
when I enable this and tries to modify following css in variables block
`.day-success {
    background-color: #47d337;
}`


it shows 

<img width="850" alt="image" src="https://user-images.githubusercontent.com/783547/182588777-6e7489d1-a97c-47b7-8038-3cee04825f0d.png">


I am using following cmd from wiki to start the docker container

`docker run -d  -p 8080:8080  -v /mydir/statping:/app   --restart always  adamboutcher/statping-ng`

and it seemed there is no box for index.scss 

and this is resolved until I manually copy the index.scss into the directory, maybe this is a bug there?

**Comment by sndrr** [comment]
Could be, this was also mentioned in [issue 600 in the original statping](https://github.com/statping/statping/issues/600).

**Comment by adamboutcher** [comment]
> Could be, this was also mentioned in [issue 600 in the original statping](https://github.com/statping/statping/issues/600).
Yes, also this https://github.com/statping-ng/statping-ng/issues/132

We are aware and intend to fix it.

**Comment by lcorr8** [comment]
@adamboutcher any updates on this?

Or has anybody actually gotten this to work with some workaround?

**Comment by radicand** [comment]
> Or has anybody actually gotten this to work with some workaround?

I worked around this with the workaround mentioned in the issue noted above and baking it into a new image (volume mounts would work fine too).
